Cargo.toml

```diff
@@ -14,4 +14,5 @@ doctest = false
 test_lib = []
 
 [dependencies]
+chrono = "0.4.31"
```

Comment:
I'm a bit reluctant to add dependencies on the template-level, mainly because we currently can't compile them in advance and the `cargo` commands hide the template compilation output.

Task for myself: need to check `chrono`'s overhead and see if we need to do something about it.